lib/checks/canMultiplyLikeTermPolynomialNodes.js

```diff
@@ -13,7 +13,6 @@ function canMultiplyLikeTermPolynomialNodes(node) {
   if (args.length === 1) {
     return false;
   }
-
```

Comment:
go to https://github.com/socraticorg/mathsteps/pull/153/files and look for all the remaining lines that need to be added back in